include/swift/SIL/LinearLifetimeChecker.h

```diff
@@ -50,6 +50,7 @@ class LinearLifetimeChecker {
   class ErrorBuilder;
 
 private:
+  friend class ReborrowVerifier;
```

Comment:
Rather than doing this, can you expose a higher level API on LinearLifetimeChecker? Maybe an asserting variant of checkUses?

Reply:
Am doing this to pass `LinearLifetimeChecker::ErrorBuilder` with continuous error number with a common `errorMessageCounter`. I would have used the public `LinearLifetimeChecker ::validateLifetime` but would loose the nice error numbering.
What do you think about exposing a version of  `LinearLifetimeChecker ::validateLifetime`  that has an errorBuilder parameter ?